pkg/inspector/image-inspector.go

```diff
@@ -467,3 +414,153 @@ func createOutputDir(dirName string, tempName string) (string, error) {
 	}
 	return dirName, nil
 }
+
+// pullExtractAndInspectImage will use containers/image library to pull extract and get the image to be scanned
+// and will inspect it for its metadata.
+// Assumes the image to be pulled is with a transport prefix
+// or if not attempts to add the "docker://" prefix to the image.
+func (i *defaultImageInspector) pullExtractAndInspectImage() (*types.ImageInspectInfo, digest.Digest, error) {
+
+	policy, err := signature.NewPolicyFromBytes([]byte(DEFAULT_SIGN_POLICY))
```

Comment:
This could be a bit more type-safe, and having to handle `err != nil`, with
```go
policy := &signature.Policy{Default: []signature.PolicyRequirement{signature.NewPRInsecureAcceptAnything()}}
```